media/js/firefox/new/scene1.js

```diff
@@ -120,4 +120,19 @@
         initOtherPlatformsModal();
     }
 
+    /**
+     * If user is on iOS and has followed an adjust.com link, redirect to
+     * the App Store.
+     * https://github.com/mozilla/bedrock/issues/5629
+     */
+    // TODO: is this a solid enough check?
+    if (window.site.platform === 'ios' && /open\-app\-store=true/.test(window.location.search)) {
+        var appStoreURL = $('.os_ios a').attr('href');
+
+        if (appStoreURL) {
+            // TODO: do we need a dataLayer call here?
```

Comment:
This could be a good idea if it may give us an idea of the drop-off rate. Could we attach some query params to the app store link maybe to attribute the referral?

Reply:
I don't _think_ we need any extra query params here, as adjust will know (with pretty high accuracy) which adjust.com link the visitor is coming from.